Ruddarr/Utilities/SimpleError.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+struct SimpleError: LocalizedError {
```

Comment:
Simple way to create an error we'll only show to the user but don't need any extra logic (no switching on the error type etc.). Suitable for most custom errors.

Reply:
Thanks, I'll just rename it.